libs/libsodium/Makefile

```diff
@@ -52,6 +52,7 @@ endef
 
 CONFIGURE_ARGS+= \
 	--disable-ssp \
+	--without-pthreads \
```

Comment:
@nbd168 What are my next steps? How I can split master and CC versions? Should I create PR for `for-15.05` branch?